### DISCUSSION TITLE
Edit the theme

### DISCUSSION THREAD

**Discussion by klunkmillan** [wprig/wprig#783] [Q&A] [answered]
so once I bundle the theme; when I need to edit that theme, say to change styling or something, do I pull it down, work in the original wprig-master, and then bundle it again and push?

**Comment by dthenley** [comment, marked as answer]
@klunkmillan yeah i would suggest making a git repo of the original files then whenever you need to make any updates to the theme you would need to pull it back down and make changes to your master repo, update the version to make sure to know that there was changes done, then rebundle your theme.

**Reply by klunkmillan** [reply]
thanks
